day_10/task_2/BUILD

```diff
@@ -3,14 +3,17 @@ cc_binary(
     srcs = ["main.cpp"],
     deps = [
         ":pipe_maze",
-        "//path_helper"
-        # "//debug_features:debug_features"],
+        "//path_helper",
+        "//debug_features:debug_features"
     ]
 )
 
 cc_library(
     name = "pipe_maze",
     hdrs = ["pipe_maze.hpp"],
     srcs = ["pipe_maze.cpp"],
-    visibility = ["//visibility:public"]
+    visibility = ["//visibility:public"],
+    deps = [
+        "//debug_features:debug_features"
+    ]
```

Comment:
debug features should be removed before the merge
```suggestion
```

Reply:
done